bridge/client/app/_services/data.service.ts

```diff
@@ -27,6 +29,7 @@ export class DataService {
   protected _traces = new BehaviorSubject<Trace[]>(null);
   protected _openApprovals = new BehaviorSubject<Trace[]>([]);
   protected _keptnInfo = new BehaviorSubject<any>(null);
+  public _remediationsUpdated: Subject<void> = new Subject<void>();
```

Comment:
I agree, I would create a `get openRemediations()` which returns an Observable.
I know that we won't be using it here, but I would also emit the updated remediations with `.next()`. At some point it might be useful, in case that the component doesn't subscribe to the project, but only to the remediations.

Reply:
I just refactored the implementation. Now all changed deployments are returned. Deployments are returned instead of remediations because the remediation config of a stage of a deployments may change.